ami-build/agent-setup.sh

```diff
@@ -51,7 +51,7 @@ sudo /srv/azure-pipelines/bin/installdependencies.sh
 
 sudo -u azure-pipelines /bin/bash -c 'mkdir -p /home/azure-pipelines/.ssh && touch /home/azure-pipelines/.ssh/known_hosts'
 sudo -u azure-pipelines /bin/bash -c 'ssh-keyscan github.com | tee /home/azure-pipelines/.ssh/known_hosts'
-sudo -u azure-pipelines /bin/bash -c 'ssh-keygen -l -f /home/azure-pipelines/.ssh/known_hosts | grep github.com | grep "SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8"'
+sudo -u azure-pipelines /bin/bash -c 'ssh-keygen -l -f /home/azure-pipelines/.ssh/known_hosts | grep github.com | grep "SHA256:uNiVztks"'
```

Comment:
What does this line actually achieve? Just print out the key? Why do we need this?

Reply:
its just checking the key against a known good hardcoded one

Reply:
todo: add explanatory comment in follow up